src/main/java/org/jenkinsci/plugins/codesonar/CodeSonarAlertCounter.java

```diff
@@ -0,0 +1,41 @@
+package org.jenkinsci.plugins.codesonar;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+/**
+ * @author aseno
+ *
+ */
+public class CodeSonarAlertCounter {
```

Comment:
I think this class `CodeSonarAlertCounter` class should live in either the "models" or in the "api" subpackage.  It should live in "models" only if other model classes are using it.  It appears that the `CodeSonarAnalysisData` model class uses it, but I am not sure the `CodeSonarAnalysisData` class is used.  If no "models" need to use this alert counter class, then it should probably live in the "api" subpackage.